subcommand/envoy-entrypoint/command_unix_test.go

```diff
@@ -111,7 +106,7 @@ func TestRun(t *testing.T) {
 				signal.Stop(sigs)
 			})
 
-			ecsMetaRequestCount := 0
```

Comment:
This I just made atomic. You need to use `atomic.Load*` and `atomic.Add*` functions though.